cypress/utils/dashboards/index.d.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+declare namespace Cypress {
```

Comment:
I like this. We should add ts support to all the utilities, which can allow proper autocomplete and type check. I think we used to have in ftr repo.

Reply:
yeah i saw them more frequently in the ftr repo. I didn't add every single one here, i was mainly adding the ones that I was using just for my development benefit (IDE auto-complete) and decided to commit it to this PR. it would be great if later we can have full TS support for all of our utilities